### ISSUE TITLE
No module named numpy

### ISSUE THREAD

**Issue by ericearl** [DCAN-Labs/abcd-hcp-pipeline#3]
Error in PreFreeSurfer:

```
user@server:/projects/ABCD/docker_pull_test$ tail output/sub-ANATFUNC/ses-baselineYear1Arm1/logs/PreFreeSurfer/PreFreeSurfer.out
Mon Feb 25 00:18:48 UTC 2019 - PreFreeSurferPipeline.sh - NOT PERFORMING GRADIENT DISTORTION CORRECTION
Mon Feb 25 01:46:27 UTC 2019 - PreFreeSurferPipeline.sh - Not Averaging T1w Images
Mon Feb 25 01:46:27 UTC 2019 - PreFreeSurferPipeline.sh - ONLY ONE AVERAGE FOUND: COPYING
Mon Feb 25 01:48:36 UTC 2019 - PreFreeSurferPipeline.sh - Aligning T1w image to 0.7mm MNI T1wTemplate to create native volume space
Mon Feb 25 01:48:36 UTC 2019 - PreFreeSurferPipeline.sh - mkdir -p /output/sub-ANATFUNC/ses-baselineYear1Arm1/files/T1w/ACPCAlignment

 START: ACPCAlignment
Final FOV is:
0.000000 176.000000 0.000000 256.000000 69.000000 150.000000

user@server:/projects/ABCD/docker_pull_test$ cat output/sub-ANATFUNC/ses-baselineYear1Arm1/logs/PreFreeSurfer/PreFreeSurfer.err
Traceback (most recent call last):
  File "/opt/fsl/bin/aff2rigid", line 76, in <module>
    from numpy import *
ImportError: No module named numpy
```

**Comment by LiquidPrototype** [comment]
Have you tried pip install numpy?

**Comment by dasturge** [comment]
fixed by 80611a8
